mlir/unittests/programs/qco_programs.cpp

```
@@ -2074,6 +2074,20 @@ void ifElse(QCOProgramBuilder& b) {
       });
 }
 
+void ifOneQubitOneTensor(QCOProgramBuilder& b) {
```

Cppcheck (2.20.0)

[style] The function 'ifOneQubitOneTensor' is never used.

(unusedFunction)